### ISSUE TITLE
Geopackage rendering issues

### ISSUE THREAD

**Issue by unaris** [NASAWorldWind/WorldWindAndroid#112]
The example Geopkg works just fine, but some of the other examples online are not rendering correctly.  The two entries from Luciad listed here: http://www.geopackage.org/#sampledata both fail, but for different reasons.

The Blue Marble example loads, but is rendered incorrectly with all the continents seemingly squeezed into about 40 degrees of the globe and rotated by 90 degrees (see attached screenshot).

The 50 states example does not contain imagery, only shape outlines, and this fails upon lookup of the gpkg_tile_matrix_set table.
![blue_marble_error_cropped](https://user-images.githubusercontent.com/213550/27035255-aa4dd038-4f4e-11e7-96ed-dec8bd1d9cbe.png)

**Comment by pdavidc** [comment]
WorldWindAndroid's GeoPackage support is currently limited to a strict tiling scheme. Both the failures and incorrect rendering you mention here are likely the result of an unsupported GeoPackage tiling scheme.

**Comment by unaris** [comment]
Are there plans to expand this capability?  If not, is there documentation on the currently used scheme?

**Comment by pdavidc** [comment]
Absolutely. There are plans to both document the currently supported tiling scheme in a more visible way and extend GeoPackage to support more tiling schemes.

Here's an overview of what's supported today:
- Raster GeoPackages containing PNG, JPEG or WebP tiles (or a mix of those)
- Tiles must be in the EPSG:4326 projection
- Tiling scheme must begin with two top level tiles containing the East and West hemisphere, and follow division by two thereafter
- Tiles may be sparse in the tiling scheme - entire levels or portions of a level may be absent from the GeoPackage
